Casks/d/display-pilot.rb

```diff
@@ -0,0 +1,20 @@
+cask "display-pilot" do
+  version "1.5.0.0"
+  sha256 "fedc92cf50f1f3264709b4dffc031aa456bd8f252fea03f72f36eb968586cd4b"
+
+  url "https://qspublic.s3.ap-southeast-1.amazonaws.com/qspublic/qs_app/1001/Display%20Pilot%202Setup-#{version}-release.dmg"
+  name "Display Pilot 2"
+  desc "Streamlines display control: brightness, inputs, pivot, and more"
+  homepage "https://www.benq.com/en-ap/monitor/software/display-pilot-2.html"
+
+  livecheck do
+    url "https://www.benq.com/en-ap/monitor/software/display-pilot-2/spec.html"
+    regex(%r{macOS</b><br />\nV([\d.]+)})
```

Comment:
```suggestion
    regex(%r{macOS\s*<.*?>\s*v?(\d+(?:\.\d+)+)}im)
```

This should only be as explicit as necessary to match what we want (and nothing more). This suggestion works for now and we can always revisit this in the future if it eventually matches something that we don't want.

This suggestion also brings the regex more in line with our prevailing standards (e.g., using `\s*` to account for potential whitespace before/after tags, using the standard regex for version matching, etc.).